tests/Unit/NumericalAlgorithms/Spectral/SwshTestHelpers.cpp

```diff
@@ -0,0 +1,115 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/NumericalAlgorithms/Spectral/SwshTestHelpers.hpp"
+
+#include <boost/math/special_functions/binomial.hpp>
+#include <cmath>
+#include <complex>
+
+#include "NumericalAlgorithms/Spectral/SwshTags.hpp"  // IWYU pragma: keep
+
+namespace Spectral {
+namespace Swsh {
+namespace TestHelpers {
+
+double factorial(const size_t arg) noexcept {
+  if (arg <= 1) {
+    return 1;
+  }
+  double factorial_result = 1.0;
+  for (size_t product_term = 1; product_term <= arg; ++product_term) {
+    factorial_result *= static_cast<double>(product_term);
+  }
+  return factorial_result;
+}
+
+std::complex<double> spin_weighted_spherical_harmonic(
+    const int s, const int l, const int m, const double theta,
+    const double phi) noexcept {
+  if (l + s < 0 or l - s < 0) {
+    return 0;
```

Comment:
I somehow missed this entire commit on my initial reviews, sorry. Lots of `0.` or `0.0` and `1.`, etc.